libs/deal-ticket/src/components/deal-ticket/deal-ticket-fee-details.tsx

```diff
@@ -376,3 +374,22 @@ export const DealTicketMarginDetails = ({
     </>
   );
 };
+
+const getAsset = (market: MarketFieldsFragment) => {
```

Comment:
This looks like a duplicate of the other getAsset function. Lets move it as a helper to libs/markets